tests/wpt/metadata/html/semantics/forms/form-submission-0/constructing-form-data-set.html.ini

```diff
@@ -1,7 +1,4 @@
 [constructing-form-data-set.html]
-  [FormData constructor always produces UTF-8 _charset_ value.]
-    expected: FAIL
-
   [_charset_ control sets the expected encoding name.]
```

Comment:
I wonder if it's because windows-1252 and iso-8859-1 are considered equivalent in https://encoding.spec.whatwg.org/. encoding_rs should be dealing with that for us, though, so it would be good to figure out what charset is being reported.

Reply:
Per [IRC](https://mozilla.logbot.info/servo/20191209#c16807443-c16807450), these changes are not expected to fix this subtest.